docs/conf.py

```diff
@@ -1,131 +1,70 @@
 """
+Configuration file for the Sphinx documentation builder.
+
 isort:skip_file
 """
 # flake8: NOQA: E402
+
+# -- stdlib imports ------------------------------------------------------------
 import os
 import sys
 import datetime
-from io import StringIO
 from pathlib import Path
-from docutils import nodes, statemachine
-from docutils.io import FileInput
-from docutils.parsers.rst import Directive, directives
-
-# -- Import base config from sphinx-astropy ------------------------------------
-try:
-    from sphinx_astropy.conf.v1 import *
-except ImportError:
-    print('ERROR: the documentation requires the "sphinx-astropy" package to be installed')
+from pkg_resources import get_distribution, DistributionNotFound
+
+# -- Check for dependencies ----------------------------------------------------
+
+doc_requires = get_distribution("sunpy").requires(extras=("docs",))
+missing_requirements = []
+for requirement in doc_requires:
```

Comment:
❤️ for the clear error message

Reply:
and also not 100 `try...except` blocks :rofl: